driver/pace/driver/driver.py

```diff
@@ -515,6 +517,9 @@ def exit_instead_of_build(self):
 
         self._time_run = self.config.start_time
 
+        self.safety_checker = SafetyChecker()
+        SafetyChecker.register_variable("ua", -200, 200, compute_domain_only=True)
+
```

Comment:
I think we should register some variables by default when you make `SafetyChecker`, then we can also allow custom checkers from the namelist